src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/ConfJsonWrapper.java

```diff
@@ -103,5 +111,6 @@ public void setDatastoreRcaLogDirectory(String rcaLogLocation) {
     this.analysisGraphEntryPoint = analysisGraphEntryPoint;
     this.networkQueueLength = networkQueueLength;
     this.perVertexBufferLength = perVertexBufferLength;
+    this.mutedRcaList = Arrays.asList(mutedRcas.split("\\s*,\\s*"));
```

Comment:
can we add a line of comment here ?

Reply:
Will add a comment line explaining what the regex is doing.